ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml

```diff
@@ -2246,6 +2246,26 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-gcp-upi-xpn
+- as: ibmcloud-ipi-dedicatedhost-dynamic-f28-destructive
+  cron: 37 8 3 * *
```

Comment:
pls run `ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml` to generate the cron.
```
[jianlinliu@jialiu-laptop jialiu_release]$ generate-cron-entry.sh ibmcloud-ipi-dedicatedhost-dynamic-f28-destructive openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml
cron: 38 11 23 * *
[jianlinliu@jialiu-laptop jialiu_release]$ generate-cron-entry.sh ibmcloud-ipi-dedicatedhost-static-f28 openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml
cron: 24 5 4 * *
```